src/main/cpp/hw/DragonSparkFlex.cpp

```diff
@@ -91,31 +104,14 @@ void DragonSparkFlex::SetControlConstants(int slot, const ControlData &controlIn
     }
 }
 
+void DragonSparkFlex::ConfigHWLimitSW(rev::SparkLimitSwitch::Type forwardType, rev::SparkLimitSwitch::Type reverseType)
+{
+    m_forwardType = forwardType;
+    m_reverseType = reverseType;
+}
+
```

Comment:
ConfigHWLimitSW should be deleted, since the switch type is set in the constructor. If they are changed, m_spark->GetForwardLimitSwitch and/or m_spark->GetReverseLimitSwitch need to be called again